components/prism-python.js

```diff
@@ -8,9 +8,16 @@ Prism.languages.python= {
 		pattern: /((^|\s)def[ \t]+)([a-zA-Z_][a-zA-Z0-9_]*(?=\())/g,
 		lookbehind: true
 	},
-	'keyword' : /\b(as|assert|async|await|break|class|continue|def|del|elif|else|except|exec|finally|for|from|global|if|import|in|is|lambda|pass|print|raise|return|try|while|with|yield)\b/,
+	'keyword' : /\b(as|assert|async|await|break|class|continue|def|del|elif|else|except|exec|finally|for|from|global|if|import|in|is|lambda|pass|print|raise|return|self|try|while|with|yield)\b/,
```

Comment:
Upon further review, it appears that `self` is not a keyword (which is likely why it wasn't there in the first place...).  I will correct this and commit an updated version only including the `class-name`.